src/hooks/core/push-notification/usePushNotificationHook.ts

```diff
@@ -16,6 +16,7 @@ import useUserAuthHook from '../auth/useUserAuthHook';
 import TokenStorage, {ITokenEnum} from '../../../utils/storage/custom/tokenStorage';
 import {getAnonymousUserId, getUserId} from '../../../utils/users';
 import {getMessageDetail} from '../../../service/repo/messageRepo';
+import useChatScreenHook from '../../screen/useChatScreenHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useChatScreenHook` is new in this hunk. Please ensure that the hook does not have any side effects when imported, as it could lead to unexpected behavior if it does.
```diff
+ import useChatScreenHook from '../../screen/useChatScreenHook';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->